pkgs/by-name/st/streamrip/package.nix

```
@@ -2,10 +2,8 @@
   lib,
   python3Packages,
   fetchFromGitHub,
-
   ffmpeg,
 }:
-
 python3Packages.buildPythonApplication rec {
```

There is no need in `rec` then.

```suggestion
python3Packages.buildPythonApplication {
```